src/api/youtube.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+const API_KEY = 'AIzaSyAN8BNOmhJQ8d6jPsYEx2SXDBnKusT7eNY';
```

Comment:
😱  Do not provide your api key at the source code in GIT, use it through an environment variable. Check for example [dotenv](https://www.npmjs.com/package/dotenv).